compose/README.md

```diff
@@ -15,12 +15,13 @@ git config --global submodule.recurse true
 - Java 11 (should be specified in JAVA_HOME)
 - [macOs/Linux] Android SDK downloaded via `./compose/frameworks/support/jbdeps/android-sdk/downloadAndroidSdk`
 - [Windows] Android SDK downloaded from Android Studio and specified in ANDROID_SDK_ROOT
+- [Optional, MacOS] For AppCode IDE, specify environment variable ANDROID_SDK_ROOT=[YOUR_PATH]/compose-jb/compose/frameworks/support/jbdeps/android-sdk/darwin (in ~/.zshrc)
```

Comment:
MacOS -> macOs

(to be consistent with above)

Reply:
Done